packages/bpk-svgs/gulpfile.babel.js

```diff
@@ -283,6 +283,7 @@ gulp.task('icons-font', ['icons-common'], () => {
        */
       normalize: true,
       fontHeight: 1001,
+      timestamp: 1436442578, // A static timestamp to prevent changes showing up in git, backpack's first commit!
```

Comment:
Could you please expand this comment here and also add some more description of this in commit message? It's not clear why this is needed

Reply:
It's to prevent git from thinking it's a changed file, if timestamp is not defined the file itself will have the current timestamp and during the build the  file's timestamp will change, what do you think would be a better message?

Reply:
tbh I think the message is ok for now, and it's blocking me so can we just merge @k0nserv